packages/spotlight/src/spotlight.js

```diff
@@ -199,11 +200,13 @@ const Spotlight = (function () {
 			return true;
 		}
 
+		const focusOptions = isWithinOverflowContainer(elem, containerIds) ? {preventScroll: true} : null;
+
 		let silentFocus = function () {
 			if (currentFocusedElement) {
 				currentFocusedElement.blur();
 			}
-			elem.focus();
+			elem.focus(focusOptions);
```

Comment:
I wasn't able to determine how widespread support is for `focusOptions`.  But, given we're already replacing a proprietary solution with this experimental one, I think it's 👍 .